sentences/de/_common.yaml

```diff
@@ -199,6 +199,30 @@ lists:
   shopping_list_item:
     wildcard: true
 
+  position:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  position_level:
+    values:
+      - in: (max[imale|imum]|höchste)
```

Comment:
```suggestion
      - in: (max[imal[e]|imum]|höchste)
```

Reply:
Hi @steffenrapp this sounds reasonable and should also be applied for `brightness_level` (_where thees ones has been copied from_). If it is ok for you, I would like keep these additions for a follow up PR (_than including `brightness_level`_), since they also need some more test sentences and those would blow up this PR

Reply:
Hi @mib1185 sure that's fine for me 👍

Reply:
@steffenrapp if you want, feel free to create a PR for these additions 👍